samples/system-test/topics.test.js

```diff
@@ -1,17 +1,16 @@
-/**
- * Copyright 2019, Google, LLC.
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+// Copyright 2019 Google LLC
```

Comment:
I'm mostly curious, why are we changing to this style of comment?

Reply:
I found it to be the only format that passes the test. I was hoping you would know.

Reply:
Oh weird, that is new to me. I had heard we were adding a robot for license checking but did not know it was particular about the formatting.

Just an FYI @bcoe @JustinBeckwith